.travis.yml

```diff
@@ -14,13 +14,13 @@ rvm:
   - ruby-head
 gemfile:
   - Gemfile
-  - Gemfile.rails5
+  - Gemfile.rails
```

Comment:
not sure `.rails` is as meaningful as `rails5` was ... maybe `.next` or `.upgrade` (something like `.rails_next` would do the trick also but ... I'm a lazy typist!)